### PR TITLE
feat(dashboard): Update dashboard page titles

### PR DESCRIPTION
**Pull request by tristandubbeld**
Changed the title of pages in the dashboard to better reflect the active page.

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Content/routes/Sandboxes/index.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ export const SandboxesPage = () => {
     >
       <Helmet>
         <title>
-          {currentPath.split('/').pop() || 'Dashboard'} - CodeSandbox
+          {currentPath.split('/').pop() || 'Sandboxes'} - CodeSandbox
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Content/routes/Sandboxes/index.tsx:87]: The `currentPath.split('/').pop()` results in `foldername`.

File: packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx [hunk 2]
```diff
@@ -126,7 +126,7 @@ export const RepositoriesPage = () => {
       items={itemsToShow}
     >
       <Helmet>
-        <title>{path || 'Dashboard'} - CodeSandbox</title>
+        <title>{path || 'Repositories'} - CodeSandbox</title>
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx:129]: The `path` results in `github/repoowner/repository`.

File: packages/app/src/app/pages/Dashboard/Content/routes/SyncedSandboxes/index.tsx [hunk 3]
```diff
@@ -78,7 +78,7 @@ export const SyncedSandboxesPage = () => {
       items={itemsToShow}
     >
       <Helmet>
-        <title>{param || 'Dashboard'} - CodeSandbox</title>
+        <title>{param || 'Synced sandboxes'} - CodeSandbox</title>
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/Content/routes/SyncedSandboxes/index.tsx:81]: Do these still exist @alexnm @olarclara?

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7218/commits/226eecc3fafc1fa05b51e609cdb3d759a0228c6b is at https://pr7218.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/Content/routes/SyncedSandboxes/index.tsx (hunk 3 above)
Comment (olarclara) [reply packages/app/src/app/pages/Dashboard/Content/routes/SyncedSandboxes/index.tsx:81]: @tristandubbeld yep, the CodeSandbox team has some synced sandboxes.

**Review by olarclara — APPROVED** [review]

**Review by filipelima18 — APPROVED** [review]
Looks great to me!

**Pull request merged**